tests/test_fixtures.py

```diff
@@ -30,6 +30,14 @@
             "M": 42,
             "GY": 44,
         },
+        "network": {
+            "Device Name": "EPSON0C9E89",
+            "Signal Strength": "Excellent",
+            "SSID": "The\xa0Bell\xa0Tower\xa0-\xa0IoT",
```

Comment:
I tweaked this to just compare the nbsps instead of converting to spaces. The nbsps are actually in your SSID, right?

Reply:
my SSID is `The Bell Tower - IoT`, I think the page is converting it to `The&nbsp;Bell&nbsp;Tower&nbsp;-&nbsp;IoT` so that it doesnt break lines while displaying. I think either solution here is fine, but probably keeping the conversion lets the test cases make more sense, and when you copy a test into this array, it will include normal spaces since your computer will copy NBSP as a normal space

Reply:
I originally fixed it in the parser, but reverted since it seemed possible that you actually had nbsps in your ssid. I restored the fix in the parser (committing soon).